src/formatter-registry.ts

```diff
@@ -103,25 +103,30 @@ var FORMATS_MAP: {[format: string]: (x: any, nullAlias: string) => string} = {
 
 var FormatterRegistry = {
 
-    get: (formatAlias: string, nullOrUndefinedAlias?: string) => {
-        var identity = FORMATS_MAP['_identity'];
-
-        var hasFormat = FORMATS_MAP.hasOwnProperty(formatAlias);
-        var formatter = hasFormat ? FORMATS_MAP[formatAlias] : identity;
-
-        if (hasFormat) {
-            formatter = FORMATS_MAP[formatAlias];
-        }
-
-        if (!hasFormat && formatAlias) {
-            formatter = (v) => {
-                var f: ((x) => string) = utils.isDate(v) ? d3.timeFormat(formatAlias) : d3.format(formatAlias);
-                return f(v);
-            };
-        }
-
-        if (!hasFormat && !formatAlias) {
-            formatter = identity;
+    get: (formatAlias: (x: any) => string | string, nullOrUndefinedAlias?: string) => {
```

Comment:
`(x: any) => string | string` is `(x: any) => string`.

Use `((x: any) => string) | string`